src/Access/User.h

```diff
@@ -22,6 +22,7 @@ struct User : public IAccessEntity
     RolesOrUsersSet default_roles = RolesOrUsersSet::AllTag{};
     SettingsProfileElements settings;
     RolesOrUsersSet grantees = RolesOrUsersSet::AllTag{};
+    String default_database;
```

Comment:
Can you please add a corresponding column into the `system.users` table?

Reply:
ok,i will do it